WordPress/src/main/java/org/wordpress/android/viewmodel/pages/PagesViewModel.kt

```diff
@@ -227,6 +233,10 @@ class PagesViewModel
         pageMap = pageStore.getPagesFromDb(site).associateBy { it.remoteId }
     }
 
+    private fun invalidatePages() {
```

Comment:
Let me know if this approach is okay to invalidate the list. I thought of it because it was a simple way and I also noticed that  a similar approach is being used to update a pages here 
https://github.com/wordpress-mobile/WordPress-Android/blob/47024f77e7ee718234bb402b05ddbe87957f85cd/WordPress/src/main/java/org/wordpress/android/viewmodel/pages/PagesViewModel.kt#L538

Reply:
Yeah, I think it's a valid approach. It's a bit more readable in Java as it's obvious there that we are using a custom setter. But I wouldn't worry about that too much.

Reply:
Okay! Great.